### DISCUSSION TITLE
Feature Request: MCAP CLI convert shows progress bar

### DISCUSSION THREAD

**Discussion by dclark-outrider** [foxglove/mcap#1124] [General] [answered]
I use `mcap convert <name>.bag <name>.mcap` fairly frequently with the CLI. If the bag is large the convert can take over 10 minutes easily and it's hard to know the status of the process.

Is it possible to get a progress report as the converter rips through the bag? 🙏🏻

**Comment by wkalt** [comment, marked as answer]
Thanks for the request. This has been implemented in the latest release of the tool.

**Reply by SlothFriend** [reply]
Amazing! Thank you for this great QoL improvement =)
